app/src/main/java/at/roteskreuz/stopcorona/screens/dashboard/changelog/ChangelogViewModel.kt

```diff
@@ -9,5 +10,11 @@ class ChangelogViewModel(
     private val changelogManager: ChangelogManager
 ) : ScopedViewModel(appDispatchers) {
 
-    fun getChangelogForVersion(version: String) = changelogManager.getChangelogForVersion(version)
+    fun getChangelogForVersion(version: String): Changelog? {
+        return changelogManager.getChangelogForVersion(version)
+    }
+
+    fun flagChangelogAsSeen() {
```

Comment:
Suggestion: Rename to `markChangelogAsSeen`

Reply:
done